Sprint-2/extend/format-time.js

```diff
@@ -0,0 +1,24 @@
+// This is the latest solution to the problem from the prep.
+// Your task is to write tests for as many different groups of input data or edge cases as you can, and fix any bugs you find.
+
+function formatAs12HourClock(time) {
+  const hours = Number(time.slice(0, 2));
+  if (hours > 12) {
+    return `${hours - 12}:00 pm`;
+  }
+  return `${time} am`;
+}
+
+const currentOutput = formatAs12HourClock("08:00");
+const targetOutput = "08:00 am";
+console.assert(
+  currentOutput === targetOutput,
+  `current output: ${currentOutput}, target output: ${targetOutput}`
+);
+
+const currentOutput2 = formatAs12HourClock("23:00");
+const targetOutput2 = "11:00 pm";
+console.assert(
+  currentOutput2 === targetOutput2,
+  `current output: ${currentOutput2}, target output: ${targetOutput2}`
+);
```

Comment:
Why not try setting up more test cases, including edge test cases like "00:00"? What do you think
`formatAs12HourClock("00:00")` and `formatAs12HourClock("12:01")` should return?